controllers/telegram_handlers.py

```diff
@@ -272,21 +272,29 @@ async def get_exam_schedule_handler(update: Update, context: ContextTypes.DEFAUL
         )
 
         response = await get_exam_schedule(user_id)
-        if response is None:
+
+        if response == "not_logged":
+            await context.bot.send_message(
+            chat_id=user_id,
+            text="There was an error, maybe you are not logged in. Use /login {amizone_id} {password} to login.",
```

Comment:
Since ``"not_logged"`` determines that a user is not logged in. We can replace
``maybe you are not logged in.`` with just ``You are not logged in. Use...``